### DISCUSSION TITLE
Is Nextra 2 (beta 41) supported on Next 13.0.3

### DISCUSSION THREAD

**Discussion by jledgett** [shuding/nextra#976] [General]
I am getting stuck on 'Creating an optimized production build' when I run `npm run build` with Next 13? 

Is Next 13 / Nextra 2 not a supported combo yet?

**Comment by jerembdn** [comment]
I don't think yet, there is a related issue here so you can follow this or fix if you find anything :)
https://github.com/shuding/nextra/issues/935
